tests/scripts/turboquant4_hf_cache_eval.py

```
@@ -0,0 +1,571 @@
+    if hasattr(past_key_values, "layers"):
+        for layer_index, layer_cache in enumerate(past_key_values.layers):
+            if not getattr(layer_cache, "is_initialized", True):
+                new_seq_lens.append(0)
+                continue
+            key = getattr(layer_cache, "keys", None)
+            value = getattr(layer_cache, "values", None)
+            if key is None or value is None:
+                raise TypeError(
+                    "TurboQuant4 HF cache eval cannot find keys/values on "
+                    f"cache layer {layer_index}."
+            )
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)